package.json

```diff
@@ -76,6 +77,7 @@
     "sinon-sandbox": "^2.0.5"
   },
   "peerDependencies": {
+    "@babel/runtime": "^7.0.0",
```

Comment:
@ljharb Now the versions of `@babel/runtime` are set to `^7.0.0` and they match! PTAL 🙏